src/src/modules/aadhaar_kyc/aadhaar_kyc.service.ts

```diff
@@ -175,6 +175,65 @@ export class AadhaarKycService {
 		} catch ({ response, message }) {
 			console.log('Error in getting OKYC status', message);
 
+			return resp.status(response?.status ? response?.status : 500).send({
+				success: false,
+				message: response?.data?.error?.detail
+					? response?.data?.error?.detail
+					: message,
+				code: response?.data?.error?.code
+					? response?.data?.error?.code
+					: 500,
+
+				data: {},
+			});
+		}
+	}
+	public async okyc2AadhaarVerification(body, req, resp) {
+		const url = `${process.env.AADHAAR_OKYC2_API_URL}`;
+		const data = { ...body };
+		try {
+			const result = await this.httpService
+				.post(url, data, this.commonRequestConfig)
+				.toPromise();
+
+			return resp.status(200).json({
+				success: true,
+				message: 'OKYC Request Successfully Verified!',
+				data: result.data,
+			});
+		} catch ({ response, message }) {
+			console.log('Error in completing OKYC', message);
+
+			return resp.status(response?.status ? response?.status : 500).send({
+				success: false,
+				message: response?.data?.error?.detail
+					? response?.data?.error?.detail
+					: message,
+
+				data: {
+					code: response?.data?.error?.code
+						? response?.data?.error?.code
+						: 500,
+				},
+			});
+		}
+	}
+	public async getOkyc2AadhaarVerification(id, req, resp) {
```

Comment:
add blank line at the start of new method

Reply:
added as per comment